packages/story-editor/src/components/panels/design/textAccessibility/textAccessibility.js

```diff
@@ -0,0 +1,168 @@
+  useEffect(() => {
+    // We want to get the text tags for the elements if not already defined
+    const textTags = getTextElementTagNames(selectedElements);
+    // Then combine the elements with their associated tag
+    const newElements = combineElementsWithTags(selectedElements, textTags);
+    // And then set them into component state
+    setSelectedTextElements(newElements);
+    setCurrentTags(Array.from(textTags.values()));
+  }, [selectedElements, setSelectedTextElements]);
+
+  if (!showSemanticHeadings) {
+    return null;
+  }
+
+  // Map all types of tag names in the selected elements
+  // and then convert to an Array for usage
+  const handleChange = (evt, value) => {
+    trackClick(evt, 'click-headings-docs');
+    selectedElements.map(() => {
+      return pushUpdate({ tagName: value });
+    });
+    // Update the selected text elements with the
+    // value chosen in the text accessibility dropdown
+    const newTags = getTextElementTagNames(selectedTextElements, value);
+    setCurrentTags(Array.from(newTags.values()));
+  };
+
+  // Check if tags match for selected item in dropdown
+  // If they don't match, we want to show 'Mixed'
+  // However, if they match, show the matching value
+  const currentValue =
+    new Set(currentTags || []).size === 1 ? currentTags[0] : MULTIPLE_VALUE;
+
+  const isIndeterminate = MULTIPLE_VALUE === currentValue;
+
+  const tagName = getCommonValue(selectedElements, 'tagName', 'auto');
+
+  const selectedValue = 'auto' === tagName ? 'auto' : currentValue;
+
+  const options = [
+    {
+      label:
+        'auto' === selectedValue
+          ? sprintf(
+              /* translators: %s: heading level. */
+              __('Automatic (%s)', 'web-stories'),
+              HEADING_LEVELS[currentValue]
+            )
+          : __('Automatic', 'web-stories'),
+      value: 'auto',
+    },
+    {
+      label: HEADING_LEVELS.h1,
+      value: 'h1',
+    },
+    {
+      label: HEADING_LEVELS.h2,
+      value: 'h2',
+    },
+    {
+      label: HEADING_LEVELS.h3,
+      value: 'h3',
+    },
+    {
+      label: HEADING_LEVELS.p,
+      value: 'p',
+    },
+  ];
+
+  return (
+    <SimplePanel
+      name="textAccessibility"
+      title={__('Accessibility', 'web-stories')}
+    >
+      <Row>
+        <DropDown
+          data-testid="text-accessibility-dropdown"
+          title={__('Heading Levels', 'web-stories')}
+          dropdownButtonLabel={__('Heading Levels', 'web-stories')}
+          options={options}
+          selectedValue={selectedValue}
+          placeholder={isIndeterminate ? MULTIPLE_DISPLAY_VALUE : currentValue}
+          onMenuItemClick={handleChange}
+          dropDownLabel={__('Heading Level', 'web-stories')}
+        />
+      </Row>
+      <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}>
+        <TranslateWithMarkup
+          mapping={{
+            a: (
+              <a
+                href={__('https://wp.stories.google/docs', 'web-stories')}
```

Comment:
```suggestion
                href={__('https://wp.stories.google/docs/seo/', 'web-stories')}
```

This will be the final URL.

Reply:
Awesome! Thank you!

Reply:
URL updated https://github.com/GoogleForCreators/web-stories-wp/pull/11888/commits/705ece200af422cea010cd4b6b17fe5843cddee3